gdal/ogr/ogrsf_frmts/dxf/ogrdxflayer.cpp

```diff
@@ -1116,6 +1119,10 @@ OGRFeature *OGRDXFLayer::TranslateLWPOLYLINE()
             dfBulge = CPLAtof(szLineBuf);
             break;
 
+          case 43:
```

Comment:
I'd have imagined something more generic than that. For example, handling that in the default case by having a bool return to TranslateGenericProperty() to indicate if the code if recognized or not. In case it is not, then add the tuple code=value to the list